benchmarks/mwt/setup/01-zookeeper-install/02-install-zookeeper.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install zookeeper --namespace kafka-mwt
```

Comment:
might be worth `  kubectl kudo install zookeeper --namespace kafka-mwt --wait`

Reply:
```
"kubectl kudo install kafka --namespace kafka-mwt -P ${KAFKA_PARAMETER_FILE}.yaml --wait" exceeded 30 sec timeout
```
adding `--wait` resulted in timeout errors of yesterday MWT, I will remove the wait as this should be done through assert step